tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4317,7 +4317,7 @@ This is [an example](http://example.com 'Example') link.
 
 [This link](http://example.com) has no title attr.
 
-This is [an example][id] reference-style link.
+This is [an example] [id] reference-style link.
```

Comment:
This change matches the expectation in the commonmark spec: https://spec.commonmark.org/dingus/?text=%5Bfoo%5D%20%5Bbar%5D%0A%0A%5Bbar%5D%3A%20%2Furl%20%22title%22%0A

`[an example]` and `[id]` appear as two separate shortcut reference links with a `text` node between them: https://astexplorer.net/#/gist/d69f5a227e2cbde3f43f0a639bee165c/822b531c252c162faba78e38607db73ef72886d5

Previously they would have appeared as a single `full` reference style link.

Reply:
Will this also fix https://github.com/prettier/prettier/issues/8226 probably?

Reply:
It sure does! I just pushed an update to include shortcut style links in this test case. I'll add another test case for that issue.

Reply:
Thanks. I'm link that issue to this PR then.